### ISSUE TITLE
Interface with Language Servers

### ISSUE THREAD

**Issue by titouandk** [continuedev/continue#2428]
### Validations

- [X] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

Will it be possible to interface Continue with language servers one day?

(To let the AI trigger clean refactors, by example.)

**Comment by sestinj** [comment]
absolutely! we're already doing this in a few places, like [here](https://blog.continue.dev/root-path-context-the-secret-ingredient-in-continues-autocomplete-prompt/) for autocomplete. Triggering larger refactors is definitely another good idea : )

**Comment by titouandk** [comment]
Very interesting, thanks :)
